modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/configuration/PageMemoryStorageDistributedConfigurationModule.java

```diff
@@ -44,7 +42,7 @@ public ConfigurationType type() {
     /** {@inheritDoc} */
     @Override
     public Collection<RootKey<?, ?>> rootKeys() {
-        return List.of(VolatilePageMemoryStorageEngineConfiguration.KEY, PersistentPageMemoryStorageEngineConfiguration.KEY);
+        return List.of();
```

Comment:
You forgot about other methods. Why did you left the class if there's no distributed storage configuration anymore? You could have renamed it at that's it.

Reply:
The whole class is removed